scania-digital-design-system/sdds
# Improvement/angular side menu demo

**Describe pull-request**  
Implemented side menu in angular 11, 12, 13 demo
- Collapsed state: it shows pop-over on hover when the side menu is in collapsed state 
- Responsive: hide/expand side menu on mobile view

**Solving issue**  

Fixes: [AB#1132](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1132)


**How to test**  
_Add description how to test if possible_
1. Go to Angular 11, 12, 13
2. Check in side menu
